__tests__/V5ClusterManagement.js

```diff
@@ -76,7 +76,7 @@ afterAll(() => {
 
 /************ TESTS ************/
 
-it('renders all the v5 cluster data correctly with 0 nodes ready', async () => {
+it('renders all the v5 cluster data correctly', async () => {
```

Comment:
I was going to fix the test, but there was nothing to be fixed: the name was wrong